oximeter/db/src/model.rs

```diff
@@ -647,267 +665,412 @@ pub(crate) fn unroll_measurement_row(sample: &Sample) -> (String, String) {
     let timeseries_name = sample.timeseries_name.clone();
     let timeseries_key = crate::timeseries_key(sample);
     let measurement = &sample.measurement;
+    unroll_measurement_row_impl(timeseries_name, timeseries_key, measurement)
+}
+
+pub(crate) fn unroll_measurement_row_impl(
+    timeseries_name: String,
+    timeseries_key: TimeseriesKey,
+    measurement: &Measurement,
+) -> (String, String) {
```

Comment:
Nit: Could we provide some documentation on the meaning of the return type? `(String, String)` is a little opqaue